ci/apache/run_integration_tests.sh

```diff
@@ -10,7 +10,8 @@ hil-admin create-admin-user $HIL_USERNAME $HIL_PASSWORD
 cd $TRAVIS_BUILD_DIR
 
 # Test commands
-py.test --cov=hil --cov-append tests/integration/cli.py
+py.test --cov=hil --cov-append tests/integration/cli.py \
```

Comment:
I don't think this is the correct way to add the test coverage to coveralls, since CLI reports 0 coverage as well.

Reply:
So rather than Coveralls, I think it's pytest-cov's fault.  The coverage reported in Travis comes from that. That's where we should start.